var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -85,7 +85,7 @@ def flag_handler(self, name, flags):
             ('x86_64',): ('sse', 'sse2', 'avx', 'avx2', 'avx512',
                           'avx-128-fma', 'kcvi'),
             ('ppc', 'ppc64le', 'power7'): ('altivec', 'vsx'),
-            ('arm',): ('neon',)
+            ('arm','arm_thunderx2'): ('neon',)
```

Comment:
Hi @floquet, the style checker is complaining because it wants a space between the "," and `'arm_thunderx2'`, like

```('arm', 'arm_thunderx2')```

Could you update that? Once that is sorted (and the CI passes) this will be good to go.

Reply:
Sorry, I think neon don't work on aarch64.
But this was wrong.
Please change line 88 to
    (aarch64, ): (neon,)

On cray target, I treate to line 91 (I don't know all cray platforms).
If this change is accepted, the x86 simd (for example avx)  is allows on the cray ARM processor machine.
Please change to line 92 to 94.